backup/queries_statistics.go

```diff
@@ -81,10 +80,9 @@ func GetAttributeStatistics(connectionPool *dbconn.DBConn, tables []Table, proce
 	SELECT c.oid,
 		quote_ident(n.nspname) AS schema,
 		quote_ident(c.relname) AS table,
-		quote_ident(a.attname) AS attname,
+		a.attname,
```

Comment:
Why do we need this change?

Reply:
> Why do we need this change?

It is needed for cases when a column has quote or space symbols. `quote_ident` adds `"` for such columns, and it prevents getting attnum from pg_attribute. For ex:
```
postgres=# create table t("a'1" int);

postgres=# select quote_ident(attname) from pg_attribute where attrelid = 'public.t'::regclass::oid and attnum > 0;
 quote_ident 
-------------
 "a'1"
(1 row)

-- but to get attnum by the name we need to compare with 'a''1', not with '"a''1"'
postgres=# select attnum from pg_attribute where attrelid = 'public.t'::regclass::oid and attname='a''1';
 attnum 
--------
      1
(1 row)

```